src/main/java/org/example/backendspringbootdomainmodelsassignment/controllers/TelevisionController.java

```diff
@@ -0,0 +1,80 @@
+package org.example.backendspringbootdomainmodelsassignment.controllers;
+
+import org.example.backendspringbootdomainmodelsassignment.models.Television;
+import org.example.backendspringbootdomainmodelsassignment.repositories.TelevisionRepository;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/televisions")
+public class TelevisionController {
+    private final TelevisionRepository repos;
+
+    public TelevisionController(TelevisionRepository repos) {
+        this.repos = repos;
+    }
+
+    @PostMapping
+    public ResponseEntity<Television> createTelevision(@RequestBody Television television) {
+        this.repos.save(television);
+        return ResponseEntity.ok(television);
+    }
+
+    @GetMapping
+    public ResponseEntity<List<Television>> getAllTelevisions() {
+        if(repos.findAll().isEmpty()) return new ResponseEntity<>(HttpStatus.NOT_FOUND);
```

Comment:
Ik snap dat je dit doet, helaas is dit niet de conventie, als er geen televisies in de lijst zijn mag je gewoon de lege lijst returnen 

Wat hier ook gebeurt is dat je 2x een database call maakt, in een kleine applicatie is dit minder erg maar stel je voor dat er 1000 mensen zijn die deze call doen per uur, dat betekent dat je 2000x een call moet verwerken naar de database ipv 1000x  :) Dus het is dan netter om net als op regel 35 de waarde tussentijds op te slaan. en daarmee verder te gaan.